tests/test_oracle_grounded_sim.py

```
@@ -0,0 +1,577 @@
+
+    def test_the_update_is_applied_and_the_arithmetic_closes(self):
+        modulators = sim.run_critic(
+            {"expected_value": 0.2, "received_reward": 0.9}, self.critic_config
+        )
+        measured = sim.run_plasticity(
+            self.weights, self.pre, modulators, self.plasticity_config
+        )
+        self.assertTrue(measured["update_applied"])
+        for before, after, delta in zip(
+            measured["weights_before"], measured["weights_after"], measured["weight_deltas"]
+        ):
```

Ruff (0.16.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)